src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -253,7 +253,7 @@ private void InitializeCore()
             DefineFullCategoryNames(LibraryRootCategories, "");
             InsertClassesIntoTree(LibraryRootCategories);
 
-            ChangeRootCategoryExpandState(BuiltinNodeCategories.GEOMETRY, true);
+            ChangeRootCategoryExpandState(BuiltinNodeCategories.GEOMETRY_CATEGORY, true);
```

Comment:
Agree, it's a little confusing.

ExportToSAT used `GEOMETRY` as well as SearchViewModel. `GEOMETRY` was the whole category Geometry.
But in category Geometry there is one class Geometry.

And appropriate name for geometry as category is `GEOMETRY_CATEGORY` and for geometry as class - `GEOMETRY`. So, ExportToSAT still uses variable `GEOMETRY`, which now is class. And SearchVM will use `GEOMETRY_CATEGORY`.